build/CoreText/lib/CoreTextLib.vcxproj

```diff
@@ -60,6 +60,10 @@
     <ClangCompile>
       <IncludePaths>$(StarboardBasePath)\Frameworks\include;$(StarboardBasePath)\include\xplat;$(StarboardBasePath)\deps\prebuilt\include;$(StarboardBasePath)\deps\prebuilt\include\freetype;</IncludePaths>
       <AdditionalOptions>"-DCORETEXT_IMPEXP= " %(AdditionalOptions)</AdditionalOptions>
+      <OtherCFlags>
+      </OtherCFlags>
+      <OtherCPlusPlusFlags>
+      </OtherCPlusPlusFlags>
```

Comment:
I have no clue how that got happened

---
In reply to: [79678864](https://github.com/Microsoft/WinObjC/pull/989#discussion_r79678864) [](ancestors = 79678864)